tools/lint.py

```diff
@@ -89,7 +90,20 @@ def run_cython_lint(files):
     return res.returncode, res.stdout
 
 
+def check_ruff_version():
+    min_version = packaging.version.parse('0.0.292')
```

Comment:
I thought I was having issues even with that older version of `ruff`, but maybe I'm mistaken.

Reply:
Very possible since it is quite an old version now, let me know if you check

Reply:
@tylerjereddy any further thoughts w.r.t whether this is the correct minimum version?